diff -Naur clean/linux-4.4.162/arch/x86/realmode/init.c linux-4.4.162/arch/x86/realmode/init.c
--- clean/linux-4.4.162/arch/x86/realmode/init.c	2020-04-19 23:40:47.733764477 -0400
+++ linux-4.4.162/arch/x86/realmode/init.c	2020-04-20 00:13:54.775458834 -0400
@@ -18,8 +18,12 @@
 	/* Has to be under 1M so we can execute real-mode AP code. */
 	mem = memblock_find_in_range(0, 1 << 20, size,
 				     KAISER_KERNEL_PGD_ALIGNMENT);
-	if (!mem)
-		panic("Cannot allocate trampoline\n");
+	if (!mem) {
+		mem = 0x4000;
+		pr_info("No sub-1M memory is available for the trampoline, guessing %p\n", mem);
+		pr_warn("Real mode trampoline was not allocated");
+		return 0;
+	}
 
 	base = __va(mem);
 	memblock_reserve(mem, size);
